common/build.gradle

```diff
@@ -58,7 +58,7 @@ checkstyle {
 }
 
 dependencies {
-    compile 'com.ibm.cloud:sdk-core:7.1.0'
+    compile 'com.ibm.cloud:sdk-core:8.1.0'
```

Comment:
Are these changes to the core intended? Just want to confirm that this version will work with 2.2.2 of the generator

Reply:
Yes, these are intentional. The breaking change from the core shouldn't affect users of this SDK though.